notebooks/WatsonNLP_Demo.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>I would leave this uncommented. Showing the length and complexity of the JSON data is an important part of the message of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>